### PR TITLE
Update licence to include both MIT and CC-BY

### LINKED ISSUE #718: (How) Does MIT license apply to book contents?
### Summary

Hi all, I'm putting together a talk on reproducibility, and in particular I want to use the graphic [here](https://the-turing-way.netlify.com/reproducibility/03/definitions.html#the-turing-way-definition-of-reproducibility) and appropriately cite it. The only license I can find quickly (admittedly a 5 minute search) is the [MIT License](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE), which is a software license, which I'm not sure how to interpret in the context of what is mostly non-executable content.

Is there guidance on reusing the contents of The Turing Way in other contexts? My inclination is simply to say something like:

> From [The Turing Way, Ch. 2](https://the-turing-way.netlify.com/reproducibility/03/definitions.html#the-turing-way-definition-of-reproducibility). License: [MIT](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE). Accessed 15 October 2019.

If the guidance exists, it would be nice if it were more prominent in the document itself, e.g., in the first or (near) final chapter.

And, assuming that the MIT license does apply to the textual contents and not just the code, I might consider a second, less software-oriented license for the text. (Though I can see that two licenses might induce more confusion than they relieve...)

### What needs to be done?

- [ ] Guidance on citation
- [ ] Optional: Add guidance to document

### Who can help?

* Anybody, likely.

---

### PR DESCRIPTION
**Pull request by KirstieJane**
### Summary

Fixes #718 

This repository contains some software code to build and host the book (as a jupyter book) on netlify but also a lot of written content that should be available under a creative commons license.

This PR updates the licence file to clarify that point.

### List of changes proposed in this PR (pull-request)

* Make licence file link to both CC-BY and MIT licence text

### What should a reviewer concentrate their feedback on?

- [ ] Does this seem the best balance of the two needs in one repository? We tried having two repositories in the past and it was a 🔥 hot 🔥 mess 🔥 becuase the issues were all over the place.
- [ ] Everything looks ok (wrt typos etc)?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: @effigies

### CODE REVIEW

**Comment by KirstieJane** [comment]
Tagging @effigies, @emdupre and @choldgraf as you might have some thoughts on this mix of licences?

**Comment by effigies** [comment]
A CC license for contents and MIT for software seems completely reasonable. Thanks for pointing me to the Zenodo (and those other issues... I swear I searched). 

A few suggestions:
* Rename to `LICENSE.md`, so that it renders?
* Add a link to the Zenodo, if that's the preferred citable object.

Possibly out-of-scope:
* Adding this or similar text to the actual document, so that people who are more familiar with books and journal articles than GitHub repositories.

**Comment by KirstieJane** [comment]
Thanks @effigies! I think I was renaming to markdown as you were typing!

Because I have the #384 PR open I'll leave the citing to that (or one of the other million issues) if that's ok. Both of your points are really important...its the author lists that are making it somewhat slow - I don't want to mess anything up by going to fast.

Hopefully I'll have some time at MozFest next week to work on the citation info 🤞

**Comment by effigies** [comment]
Sounds good.

**Comment by emdupre** [comment]
> Does this seem the best balance of the two needs in one repository? We tried having two repositories in the past and it was a fire hot fire mess fire becuase the issues were all over the place.

I think this is a great balance ! I looked into it a bit and it seems that people have all sorts of methods for incorporating two licenses, but after my (very brief) search I'd agree that just having them in one file and specifying what applies where makes the most sense. If there are specific objects that arise as points of confusion (maybe some well-documented code-hybrid ?),  we can use those to delineate the two licenses even more finely. But what you have now makes perfect sense to me !

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# LICENSE
+
+The Turing Way Community make all their materials publicly available under open source licenses.
+
+The process documents and data are made available under a CC-BY license.
+Software are made available under an MIT license.
+
+The text below is re-used, under a CC-BY license from The Carpentries community materials.
```
Comment (emdupre) [inline LICENSE.md]: This confused me a bit. One light suggestion could be:

```suggestion
The  license text listed below (describing both CC-BY and MIT licenses as well as their usage in The Turing Way) is re-used under a CC-BY license from The Carpentries community materials.
```

Or something like that !

**Comment by KirstieJane** [comment]
Thanks @emdupre for thinking through this - and thank you for the grammar fixes!

If you can review and approve I’ll merge this in. We can fix forwards if @choldgraf recommends something different 💖💖

**Comment by KirstieJane** [comment]
@allcontributors bot! Please add @effigies for ideas!

**Comment by KirstieJane** [comment]
@allcontributors bot - please add @emdupre for reviewing

**Comment by KirstieJane** [comment]
Sorry @allcontributors bot - @emdupre and review

**Comment by KirstieJane** [comment]
😭 last try. @allcontributors - please add @emdupre for review

**Comment by KirstieJane** [comment]
🚀🚀🚀

**Comment by choldgraf** [comment]
FWIW, I agree with others in this post that CC-BY is a good license for "content" and MIT is a good open license for "code", those are the two general licenses I usually recommend :-)

**Comment by KirstieJane** [comment]
MERGED! Thank you all ✨

**Comment by rainsworth** [comment]
Sorry, trying again. @allcontributors - please add @emdupre for review

**Pull request merged**